proposals/lip-0051.md

```diff
@@ -503,242 +641,1892 @@ crossChainTransferMessageParams = {
+            "dataType": "uint32",
+            "fieldNumber": 5
+        }
+    }
+}
+```
+
+#### unlock
+
+This event has `name = EVENT_NAME_UNLOCK`. This event is emitted when the unlock function is called.
+
+##### Topics
+
+* `address`: the address of the account from which to unlock the tokens.
+* `tokenID`: the token ID of the token to be unlocked.
+
+##### Data
+
+```java
+unlockEventDataSchema = {
+    "type": "object",
+    "required" = [
+        "address",
+        "module", 
+        "tokenID", 
+        "amount", 
+        "result"
+    ],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+        "module": {
+            "dataType": "string",
+            "minLength": MIN_MODULE_NAME_LENGTH, 
+            "maxLength": MAX_MODULE_NAME_LENGTH,
+            "fieldNumber": 2
+        },
+        "tokenID": {
+            "dataType": "bytes",
+            "length": TOKEN_ID_LENGTH,
+            "fieldNumber": 3
+        },
+        "amount": {
+            "dataType": "uint64",
+            "fieldNumber": 4
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 5
+        }
+    }
+}
+```
+
+#### initializeToken
+
+This event has `name = EVENT_NAME_INITIALIZE_TOKEN`. This event is emitted when the initialize token function is called.
+
+##### Topics
+
+* `tokenID`: the token ID to be initialized.
+
+##### Data
+
+```java
+initializeTokenEventDataSchema = {
+    "type": "object",
+    "required" = ["tokenID", "result"],
+    "properties": {
+        "tokenID": {
+            "dataType": "bytes",
+            "length": TOKEN_ID_LENGTH,
+            "fieldNumber": 1
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 2
+        }
+    }
+}
+```
+
+#### initializeUserStore
+
+This event has `name = EVENT_NAME_INITIALIZE_USER_STORE`. This event is emitted when the initialized user store function is called.
+
+##### Topics
+
+* `address`: the address of the user store to be initialized.
+
+##### Data
+
+```java
+initUserEventDataSchema = {
+    "type": "object",
+    "required" = ["address", "tokenID", "initPayingAddress", "initializationFee" , "result"],
+    "properties": {
+        "address": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 1
+        },
+        "tokenID": {
+            "dataType": "bytes",
+            "length": TOKEN_ID_LENGTH,
+            "fieldNumber": 2
+        },
+        "initPayingAddress": {
+            "dataType": "bytes",
+            "length": ADDRESS_LENGTH,
+            "fieldNumber": 3
+        },
+        "initializationFee": {
+            "dataType": "uint64",
+            "fieldNumber": 4
+        },
+        "result": {
+            "dataType": "uint32",
+            "fieldNumber": 5
+        }
+    }
+}
+```
+
+#### initializeEscrowStore
```

Comment:
same as above, maybe better to say `initializeEscrowAccount`?

Reply:
done.